app/src/main/AndroidManifest.xml

```diff
@@ -83,6 +83,20 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/file_paths" />
         </provider>
+
+        <!-- Disable the Andriod WorkManager in favour of the org.koin.androidx.workmanager.koin.workManagerFactory -->
+        <!-- As per the Koin Work Manager instructions https://insert-koin.io/docs/reference/koin-android/workmanager -->
```

Comment:
Hmm, if WorkManager is an issue with Koin and needs an extra dependency to work, maybe `WorkManager` could be replaced with `JobScheduler`. If I remember correctly, `WorkManager` was introduced to have an API for deferred jobs work with Android API levels below 21. For Android API 21+, there is the `JobScheduler` API. Since this app now supports only API 21+, the `WorkManager` could be dropped completely. What do you think?

Reply:
I've had to read up about WorkManager a bit more to get a better understanding of what its strengths are.

This [Guide to background work](https://developer.android.com/guide/background/) being the most useful resource I've found so far.

What I understand is that `WorkManager` is a more recent than `JobScheduler`, with there being more guidance/support around using Koin with `WorkManager` than Koin with `JobScheduler`.

Reply:
Hmm okay. I may be revisiting this in the future though, WorkManager seems to be a pretty heavy dependency - over 1MB